### PR TITLE
Second step of the import of the MCH geometry

### PR DESCRIPTION
**Pull request by flodamas**
This PR is mainly a fix to the first one (#1144). Moreover, the geometry builder of the stations 1 & 2 is improved : constant values cleaned up, more detailed geometry, ... Some parts are still under investigation.

### CODE REVIEW

**Comment by flodamas** [comment]
Sorry for the long list of commits, I should have merged by branch in dev first.

File: Detectors/MUON/MCH/Simulation/src/Materials.cxx [hunk 1]
```diff
@@ -189,12 +190,17 @@ void createMaterials()
...
   float dPlastic = 1.107;
   mgr.Mixture(kModuleName, ++imat, "Plastic", aPlastic, zPlastic, dPlastic, -nPlastic, wPlastic);
-  mgr.Medium(kModuleName, Medium::Plastic, "Plastic", imat, 0, fieldType, maxField, kMaxfd, kStemax, kDeemax, kEpsil,
+  mgr.Medium(kModuleName, Medium::Plastic, "Plastic", imat, kIsUnsens, fieldType, maxField, kMaxfd, kStemax, kDeemax, kEpsil,
              kStmin);
 
-  /// Epoxy (definition taken from AliMUONCommonGeometryBuilder)
-  mgr.Material(kModuleName, ++imat, "Epoxy", 12.24, 6., 1.85, 0., 0.);
-  mgr.Medium(kModuleName, Medium::Epoxy, "Epoxy", imat, 0, fieldType, maxField, kMaxfd, kStemax, kDeemax, kEpsil,
+  /// Epoxy : C18 H19 O3 (to be confirmed)
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Simulation/src/Materials.cxx]: Hi Florian,
Not a big fan of (to be confirmed, todo, etc...) in the code. You'd better put that one(s) in JIRA instead (or in addition to... then should remember to remove the comments in the code once this is settled)

File: Detectors/MUON/MCH/Simulation/test/testGeometry.cxx [hunk 2]
```diff
@@ -136,7 +136,11 @@ BOOST_AUTO_TEST_CASE(GetTransformations)
   BOOST_REQUIRE(gGeoManager != nullptr);
 
   o2::mch::mapping::forEachDetectionElement([](int detElemId) {
-    BOOST_CHECK_NO_THROW((o2::mch::getTransformation(detElemId, *gGeoManager)));
+    auto t = o2::mch::getTransformation(detElemId, *gGeoManager);
+    Point3D<float> m;
+    t.LocalToMaster({ 0, 0, 0 }, m);
+    std::cout << "DE " << std::setw(4) << detElemId << " " << m << "\n";
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Simulation/test/testGeometry.cxx]: I guess this is a leftover, right ?

**Comment by aphecetche** [comment]
Your have changes in Framework that are not intended I guess ... Formatting issues maybe ?

File: Detectors/MUON/MCH/Simulation/test/testGeometry.cxx (hunk 2 above)
Comment (flodamas) [reply Detectors/MUON/MCH/Simulation/test/testGeometry.cxx]: Oh indeed i missed this one !

File: .gitignore [hunk 3]
```diff
@@ -61,7 +61,6 @@ compile_commands.json
 .ycm_extra_conf.py
 
 # Datafiles
-gphysi.dat
```
Comment (aphecetche) [inline .gitignore]: well, I think it's fine to indeed exclude that one : why did you removed it?

Comment (flodamas) [reply .gitignore]: At our first PR, we excluded it but It was a mistake. When I noticed it, it was too late and the PR was already merged. According to @sawenzel , it should never be excluded in principle.

**Comment by aphecetche** [comment]
well, the conflict thing should not be resolved "by hand" : the files you did not modified (or that are not relevant to this PR) should not be part of this PR at all, period ;-)

**Comment by flodamas** [comment]
Yes indeed. I will try to remove them from this PR.

**Comment by flodamas** [comment]
How can I see the reason why the "macos" check is errored ? When I click on details, it sends me to a 404 error page ...

File: .gitignore (hunk 3 above)
Comment (sawenzel) [reply .gitignore]: I think I was fine with putting it into .gitignore. Please remove the change from the PR.

**Comment by sawenzel** [comment]
@aphecetche : Do you approve this now?

**Comment by aphecetche** [comment]
Was the printout from the test removed ? 
Other than that looks good to me.

**Comment by flodamas** [comment]
Hmmm I thought I removed it this morning.

**Comment by sawenzel** [comment]
Squashed (in order to clean up the commit history and avoid merge commits). Please rebase on dev and refork to continue development.

**Pull request merged**